2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 7 Line 1_
```python
importances = model."feature_importance_property"
```
Add a comment indicating that this is the line that should be replaced.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
This was written in the cell above, but I have made it more clear.